pkgs/games/factorio/default.nix

```diff
@@ -17,7 +17,7 @@ let
   variants = {
     x64 = {
       url = "https://www.factorio.com/get-download/${version}/alpha/linux64";
-      sha256 = "1xpzrx3q678519qgjl92fxn3qv55hd188x9jp6dcfk2ljhi1gmqk";
+      sha256 = "1dc53swnlbz23czw8v3ldl4ayp7118qyja4vh1z5dsd871swz9qg";
     };
 
     x32 = {
```

Comment:
The hash below here also has to be updated, but I'm not sure how to generate the hash. Any hints welcome :)